benchmark/framework/containerd_utils.go

```diff
@@ -258,6 +261,17 @@ func (proc *ContainerdProcess) RunContainerTaskForReadyLine(
 	if err := taskDetails.task.Start(ctx); err != nil {
 		return nil, err
 	}
+	if parseFileAccessPatterns {
```

Comment:
Should this be after the container exits? Right now we'll only get partial data based on however far into the benchmark run the container is when the parser runs, right?

Reply:
The parser will read the logs from both `containerd-stderr` and `soci-snapshotter-stderr` after the tests for an image are run.  This was initially there to record the start time of the container. ( No longer getting start time here)